core/src/main/java/me/leoko/advancedban/utils/Command.java

```diff
@@ -557,6 +558,9 @@ public void execute(Object player, String[] args) {
 
 
     public TabCompleter getTabCompleter() {
+        MethodInterface mi = Universal.get().getMethods();
+        if (!mi.getBoolean(mi.getConfig(), "Use Tab Completion", true)) 
+            return new NullTabCompleter();
```

Comment:
Not that it has any actual impact on performance, but to be consistent, getter methods should only retreive attributes and not instantiate new objects. 
I would suggest creating a static final instance of a NullTabCopleter outside the method and returning it here.

Reply:
Done as per request, let me know if you'd want it differently 👍